gui-pkg-manager-lib/pkg/gui/private/by-source.rkt

```diff
@@ -393,7 +393,7 @@
         [(github) sc-install-pkg-github]
         [(file-url) sc-install-pkg-file-url]
         [(dir-url) sc-install-pkg-dir-url]
-        [else (error 'type->str "unknown type ~s\n" type)]))
+        [else (error 'type->str (string-constant install-pkg-unknown) type)]))
     
```

Comment:
I think this is an internal error, so we don't need a string constant for it.

Reply:
@rfindler - fixing this.